core/urls.py

```diff
@@ -49,8 +48,10 @@ def test_sentry(request):
         CustomRegistrationView.as_view(form_class=CustomRegistrationForm),
         name="django_registration_register",
     ),
+    # TODO more things to move to auth
```

Comment:
Ideally we should list what needs to be moved in a Tech-Debt card if we land the patch as-is